pkg/README.md

```diff
@@ -10,6 +10,12 @@
   </ul>
 </div>
 
+
+## Release Notes
+
+Version 0.3.0 requires Bazel 3.2.0 or above.
```

Comment:
This could perhaps say "Version 0.3.0 or later" to be maximally clear, but this is good as-is, though.

Reply:
Done